operator/tools/go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v4 v4.5.7
 )
 
+replace github.com/operator-framework/operator-sdk v1.24.1 => github.com/porridge/operator-sdk v1.5.1-0.20230329122232-413d752ff864
```

Comment:
Why is a fork necessary? I've looked into the change and to me at a first glance looked unrelated.